python/lsst/dax/imgserv/hashutil.py

```diff
@@ -29,10 +29,10 @@
 import hashlib
 
 class Hasher(object):
-    """ Hasher is simple wrapper to sha256 functionality.
+    """ Hasher is simple wrapper to md5 functionality.
```

Comment:
There should be some documentation here to explain why such a weak hashing function is acceptable.

Reply:
The hashing function only used internally for generating API signatures, not touching the scientific data, e.g. image or metadata, so sha256 is an overkill (bloated for length). Will comment it as such.

Reply:
Switching to sha from md5 in a smashed commit is also annoying, but is the only reason why we're changing this because we think that sha256 is too long?  (I'm not sure that crosses my bar of 'worth it'), but then again, it's already done.

Reply:
I'm going to add back the sha256 method in Hasher, w/the comment that it should be used for science data - related operations, e.g. image data or metadata matching.   For anything else, like API matching, MD5 is sufficient and more convenient, easier manual/visual validation.